apps/web/ui/analytics/top-links.tsx

```diff
@@ -43,7 +43,9 @@ export default function TopLinks() {
                         className="size-5 sm:size-5"
                       />
                     ),
-                    title: !showUrls && d["shortLink"] ? d["shortLink"] : d.url,
+                    title:
+                      (!showUrls && d["shortLink"] ? d["shortLink"] : d.url) ??
+                      "Unknown",
```

Comment:
I don't think this will affect production, but I was running into some errors when my local data didn't have some links that Tinybird had data for

Reply:
oh gotcha, yeah good catch, let's go with this just in case 👍